contribute/en/Style-and-Tone.md

```diff
@@ -97,6 +97,8 @@ Jamulus is “Free and Open Source (FOSS)” (not “free software” or “open
 
 “Country/Region” Keep in mind that some areas of the world have a controversal (political) status. If possible, be generic and remain neutral. Instead of just saying country, use “Country/Region“ or “Location“.
 
+Registration: When a Server is configured in Registered mode, it will be "listed" when successfully registered by a Directory. Note that if a Directory is full, a Registered Server may not therefore be "listed" becuase it has not been successfully "registered".
```

Comment:
So a Registered Server isn't actually registered sometimes, since _Registered_ is actually a _mode_, which here implies there's a step that might fail. So _Registered Server_ actually means _A Server That Tries to List on a Directory (And Might Fail)_. 

In my fever dream, I write: A Listed Server appears in a directory's list of servers. Using the Server UI, you can remove the server from its directory temporarily to become an Unlisted Server. If the directory is full, the server's attempt to list itself on the directory will fail, an error will appear, and the server will operate as an Unlisted Server.

Reply:
This is just the style guide for people who write docs. I was attempting to clarify the use of the terms as we currently have them (so to avoid, eg. "The Server isn't registered by the Directory"). The why/how stuff isn't relevant here.